models/modelsmodule.cpp

```diff
@@ -309,8 +303,8 @@ ModelsModule::init( SLIInterpreter* )
 
   // register all synapse models
   register_connection_model< bernoulli_synapse >( "bernoulli_synapse" );
-  register_connection_model< clopath_synapse >(
-    "clopath_synapse", default_connection_model_flags | RegisterConnectionModelFlags::REQUIRES_CLOPATH_ARCHIVING );
+  auto flags = default_connection_model_flags | RegisterConnectionModelFlags::REQUIRES_CLOPATH_ARCHIVING;
```

Comment:
Not a fan of introducing a temporary variable here to hold the flags, but if you prefer it this way, could you move the declaration to the top of the method?

Reply:
I might indeed have gone a bit too far here and have reverted to the original code.